CHANGES.md

```diff
@@ -1,5 +1,17 @@
 ## Changelog
 
+### HEAD
+ * Compatibility: allow pandas version newer than 1.0, less than 2.0
+ * Compatibility: allow numpy version newer than 1.18
+ * Build: specify which version of python/numpy/pandas to test
+ * Bugfix: resolve import error on pandas.Panel, Panel was removed in pandas 1.0
```

Comment:
It'd be nice to have issue numbers for these, but not super important.  
Once merged i'll go thru historic issues fixed by this PR.

Reply:
I've added numbers for the Panel import error, it's the only bug that had open PRs.